scripts/arcsign.pl

```diff
@@ -48,6 +50,22 @@
 	binmode STDIN;
 }
 
+my %arc_opt;
```

Comment:
Indentation looks really bad again.  Maybe it's just my UI, but it looks really inconsistent.  New editor woes?

![image](https://github.com/fastmail/mail-dkim/assets/16064731/9ecd537f-f334-4a71-8bf1-95e5aad71a5a)

Reply:
And then you fixed it in the final commit.  Why do i review one commit at a time 😆